vendor/composer/autoload_real.php

```diff
@@ -23,7 +23,7 @@ public static function getLoader()
         self::$loader = $loader = new \Composer\Autoload\ClassLoader();
         spl_autoload_unregister(array('ComposerAutoloaderInit22ddb69348c7ed922c96325249cef3d0', 'loadClassLoader'));
 
-        $useStaticLoader = PHP_VERSION_ID >= 50600 && !defined('HHVM_VERSION');
+        $useStaticLoader = PHP_VERSION_ID >= 50600 && !defined('HHVM_VERSION') && (!function_exists('zend_loader_file_encoded') || !zend_loader_file_encoded());
```

Comment:
I "cheat" and do reverse searches for "openemr_composer" in my terminal history to run this all the time:

```composer install && ./vendor/bin/phing vendor-clean && rm -rf vendor/phing && rm -rf vendor/bin/phing && composer dump-autoload -o # openemr_composer```

This is what I used to check in this PR. Is something off with the command?